### ISSUE TITLE
Index out of range error for label, sometimes...

### ISSUE THREAD

**Issue by jpecor** [adafruit/Adafruit_CircuitPython_Display_Button#5]
I am getting an index out of range error from the button_3 code in the example:

`
button_3 = Button(x=BUTTON_MARGIN*3+2*BUTTON_WIDTH, y=BUTTON_MARGIN*2+BUTTON_HEIGHT,
                  width=BUTTON_WIDTH, height=BUTTON_HEIGHT,
                  label="button3", label_font=font, label_color=0x0000FF,
                  fill_color=None, outline_color=None)
buttons.append(button_3)`

Here's the error:

`Traceback (most recent call last):
  File "code.py", line 71, in <module>
  File "adafruit_button.py", line 127, in __init__
  File "adafruit_button.py", line 140, in label
IndexError: Group index out of range`

After tinkering a bit with the code, I found that if I pass in a value for EITHER the fill_color OR outline_color, it works.  So, for example, this works:

`button_3 = Button(x=BUTTON_MARGIN*3+2*BUTTON_WIDTH, y=BUTTON_MARGIN*2+BUTTON_HEIGHT,
                  width=BUTTON_WIDTH, height=BUTTON_HEIGHT,
                  label="button3", label_font=font, label_color=0x0000FF,
                  fill_color=0x00FF00, outline_color=None)
buttons.append(button_3)`

![IMG_8025](https://user-images.githubusercontent.com/14111408/54927306-ea794380-4edf-11e9-8227-418a56485055.jpg)

I have not dug into this any deeper, yet, but wanted to post the issue in case it's more obvious to someone else.

**Comment by anecdata** [comment]
I get very similar errors. If the `outline_color` is 0x0 or None, it errors: 
```
File "adafruit_button.py", line 136, in __init__
File "adafruit_button.py", line 149, in label
IndexError: Group index out of range
```
But if `outline_color` is 0xFF0000 for example, it's fine.
[PyPortal, latest CircuitPython 03f9048 on 2019-04-19, Library Bundle 20190420]

**Comment by makermelissa** [comment]
Fixed by #7

**Comment by jpecor** [comment]
Thank you!
